src/hooks/use-authentication/index.ts

```diff
@@ -1,7 +1,13 @@
 import { useSession } from 'next-auth/react'
-import { SessionData, UserData, ValidAuthProviderId } from 'types/auth'
+import {
+	AuthErrors,
+	SessionData,
+	UserData,
+	ValidAuthProviderId,
+} from 'types/auth'
 import { UseAuthenticationOptions, UseAuthenticationResult } from './types'
 import { signInWrapper, signOutWrapper, signUp } from './helpers'
+import { useEffect } from 'react'
```

Comment:
⛏️ Can this be moved to the top of the import list?

Reply:
Done!